### ISSUE TITLE
scaladoc claims 'missing argument for option -skip-packages'

### ISSUE THREAD

**Issue by raboof** [lampepfl/dotty#14939]
## Compiler version

Tested with 3.1.0 (commandline) and 3.1.2 (from sbt)

## Minimized code

```bash
scaladoc -skip-packages foo.bar
```

## Output

```
missing argument for option -skip-packages
  scaladoc -help  gives more information
```

## Expectation

complete successfully, skipping foo.bar

**Comment by romanowski** [comment]
Hi @raboof, `scaladoc -skip-packages:foo.bar` should work and from what I know that is the only supported way for options that requires `List`.

Also, `-skip-packages` is [deprectated](https://github.com/lampepfl/dotty/blob/main/scaladoc/src/dotty/tools/scaladoc/ScaladocSettings.scala#L62-L63), please use `-skip-by-id` or `-skip-by-regex`.

**Comment by romanowski** [comment]
Yeah, I think so. I am closing the issue, for now, please let us know if the fact that old syntax is not supported is problematic.

**Comment by raboof** [comment]
> please let us know if the fact that old syntax is not supported is problematic.

It's always possible to work around it of course. Perhaps it's not that useful to keep the 'deprecated' option around at all if it can't actually be invoked the way it used to?
